tensorrt_llm/llmapi/utils.py

```
@@ -622,6 +622,49 @@ def get_numa_aware_cpu_affinity(device_id):
+                            key=lambda e: e.name):
+            if not entry.name.startswith("node"):
+                continue
+            cpulist_path = os.path.join(entry.path, "cpulist")
+            if not os.path.exists(cpulist_path):
+                continue
+            with open(cpulist_path) as f:
+                cpus = _parse_cpulist(f.read().strip())
+            if current_cpu in cpus:
+                return cpus
+    except Exception:
+        pass
```

Ruff (0.15.15)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)